apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -1341,7 +1341,7 @@ export class InstallManager {
       }
 
       if ((this._rushConfiguration.packageManagerWrapper as PnpmPackageManager).supportsResolutionStrategy) {
-        args.push('--resolution-strategy', this._rushConfiguration.pnpmOptions.resolutionStrategy);
+        args.push(`--resolution-strategy=${this._rushConfiguration.pnpmOptions.resolutionStrategy}`);
```

Comment:
Is this CLI syntax also compatible with older versions of pnpm?

Reply:
yes